src/Microsoft.Diagnostics.EventFlow.Core/Implementations/EventFlowJsonUtilities.cs

```diff
@@ -0,0 +1,24 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Diagnostics.EventFlow
+{
+    public class EventFlowJsonUtilities
+    {
+        private static readonly JsonSerializerSettings s_defaultSerializerSettings = new JsonSerializerSettings()
+        {
+            NullValueHandling = NullValueHandling.Ignore,
+            DateFormatHandling = DateFormatHandling.IsoDateFormat,
+            ReferenceLoopHandling = ReferenceLoopHandling.Ignore
```

Comment:
What does Ignore flag imply? Does it potentially allows the reference loop? Why do we choose to ignore it?

Reply:
Consider using read-only property? For example:
```
public static JsonSerializerSettings DefaultSerializerSettings { get; } = new JsonSerializerSettings(){ ... }
```

Reply:
`ReferenceLoopHandling.Ignore` will prevent reference loops from breaking serialization https://www.newtonsoft.com/json/help/html/SerializationSettings.htm#ReferenceLoopHandling 

I will follow your advice to use a read-only property

Reply:
hi guys, thanks for all the hard work, any plan for releasing these changes soon please? We are hitting a Json Serialization issue (self referencing loop) which seems to be resolved in this change set. Thanks a lot.

Reply:
oh is it called AzureMonitorLog now?

Reply:
@lnhzd Correct, AzureMonitorLog is the new name

Reply:
thanks installed AzureMonitorLog and problem solved!